platform/qemu.go

```diff
@@ -265,19 +265,30 @@ func CreateQEMUCommand(board, uuid, biosImage, consolePath, confPath, diskImageP
 			"-cpu", "host",
 			"-m", "2048",
 		}
+	case "s390x--s390x-usr":
+		qmBinary = "qemu-system-s390x"
+		qmCmd = []string{
+			"qemu-system-s390x",
+			"-machine", "s390-ccw-virtio,accel=kvm",
+			"-cpu", "host",
+			"-m", "2048",
+		}
 	default:
 		panic("host-guest combo not supported: " + combo)
 	}
 
 	qmCmd = append(qmCmd,
-		"-bios", biosImage,
 		"-smp", "1",
 		"-uuid", uuid,
 		"-display", "none",
 		"-chardev", "file,id=log,path="+consolePath,
 		"-serial", "chardev:log",
 	)
 
+	if runtime.GOARCH != "s390x" {
```

Comment:
`runtime.GOARCH` is the architecture of the host, this should be checking the `board` variable.